app/themes/base/buttons.js

```diff
@@ -398,6 +398,34 @@ export default ({
         color: colors.text.primaryDisabled,
       },
     },
+    tab: {
+      ...defaultStyles,
+      padding: `12px ${space[5]}px`,
+      fontWeight: fontWeights.medium,
+      backgroundColor: colors.blue50,
+      color: colors.white,
+      border: 0,
+      borderRadius: radii.default,
+      borderBottomRightRadius: 0,
+      borderBottomLeftRadius: 0,
+      '.icon img': {
+        filter: 'brightness(0%) saturate(100%) invert(97%) sepia(100%) saturate(0%) hue-rotate(317deg) brightness(105%) contrast(101%)', // white
```

Comment:
filters used to allow displaying the same SVG image icons with different colors, since the way they are loaded into the React components uses a data url src rather than raw svg elements that we can adjust the fill for.